pkgs/by-name/li/lianad/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lianad";
+  version = "5.0"; # keep in sync with liana
+
+  src = fetchFromGitHub {
+    owner = "wizardsardine";
+    repo = "liana";
+    rev = "v${version}";
+    hash = "sha256-RkZ2HSN7IjwN3tD0UhpMeQeqkb+Y79kSWnjJZ5KPbQk=";
+  };
+
+  cargoSha256 = "sha256-6r8G4sO/aDO5VyJbmbdFz3W2b8ZmulXJZtnz8IAwcaU=";
+
+  buildInputs = [ systemd ];
+
+  postInstall = ''
+    install -Dm0644 ./contrib/lianad_config_example.toml $out/etc/liana/config.toml
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
+    mainProgram = "lianad";
+    description = "A Bitcoin wallet leveraging on-chain timelocks for safety and recovery";
```

Comment:
```suggestion
    description = "Bitcoin wallet leveraging on-chain timelocks for safety and recovery";
```

Reply:
this is also just following the `liana` package:

https://github.com/NixOS/nixpkgs/blob/8764d898c4f365d98ef77af140b32c6396eb4e02/pkgs/by-name/li/liana/package.nix#L85